.github/workflows/publishWorkflow.yml

```diff
@@ -19,7 +19,8 @@ jobs:
 
       # Task execution
       - run: npm run validate:environment
-      - run: npm run build:only
+      - run: npm run build:vite
+      - run: npm run build:types
       # - run: npm publish
       #   env:
       #     NODE_AUTH_TOKEN: ${{ github.token }}
```

Comment:
NPM publish will get enabled in the next PR. I have to talk to infra about what's involved in pushing code to NPM proper using Babbel's `@babbel` scope

Reply:
Good point, should we rename `@lessonnine/rollbar-client.js` to `@babbel/rollbar-client.js` perhaps?
As I think `lessonnine` makes sense for our internal packages, but for external it might make more sense to go with `babbel`?

Reply:
That's exactly right, I've already changed the README to reflect that

Reply:
And I think we also need to update the [package.json](https://github.com/babbel/rollbar-client.js/pull/5/files/a16e6e8e6062eff5c8b5b0bc286d9b2118a7d987#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R2)